### PR TITLE
Add ability to package Interface with a custom startup location (also tacks on some build docs updates)

### PR DESCRIPTION
**Pull request by kasenvr**
This PR will allow you to set an environment variable on build time of `INITIAL_STARTUP_LOCATION` equal to a URL / location that you want the Interface to launch to by default. 

On first run, it will send the user to this location, it will also set the startup location to the very same and set the "Start-up Location" to "Home". The Home will be the URL specified at build time.

### CODE REVIEW

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR640/Vircadia-Alpha-PR640-5da2599_b9a25905.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR640/Vircadia-Alpha-PR640-5da2599.tar_b9a25905.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR640/Vircadia-Alpha-PR640-5da2599.tar_b9a25905.gz

**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR640/Vircadia-Alpha-PR640-5da2599_b9a25905.dmg

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR640/Vircadia-Alpha-PR640-5da2599_b9a25905.exe

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR640/Vircadia-Alpha-PR640-58c4bda.tar_d41cb0a5.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR640/Vircadia-Alpha-PR640-58c4bda.tar_d41cb0a5.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR640/Vircadia-Alpha-PR640-58c4bda_d41cb0a5.sh

**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR640/Vircadia-Alpha-PR640-58c4bda_d41cb0a5.dmg

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR640/Vircadia-Alpha-PR640-58c4bda_d41cb0a5.exe

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR640/Vircadia-Alpha-PR640-b994578.tar_60e6d99a.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR640/Vircadia-Alpha-PR640-b994578.tar_60e6d99a.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR640/Vircadia-Alpha-PR640-b994578_60e6d99a.sh

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR640/Vircadia-Alpha-PR640-b994578_60e6d99a.exe

**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR640/Vircadia-Alpha-PR640-b994578_60e6d99a.dmg

**Comment by kasenvr** [comment]
Once this is code reviewed, given that the review concludes that this is not going to mess with anything, I'll consider it mergeable since it's a build flag that is unlikely to be tested right this moment by anyone other than myself. It's also a very specific build flag so if anyone ever tries and it doesn't work, it won't be too hard to debug. :) I expect it will come into use for many situations so I will also be testing regularly and probably expanding on the functionality regardless.

File: INSTALL.md [hunk 1]
```diff
@@ -59,6 +63,12 @@ To produce an executable installer on Windows, the following are required:
 1. [Node.JS and NPM](<https://www.npmjs.com/get-npm>)
     1.  Install version 10.15.0 LTS
     
+##### Code Signing (optional)
+
+For code signing to work, you will need to set the `HF_PFX_FILE` and `HF_PFX_PASSPHRASE` environment variables globally.
```
Comment (ctrlaltdavid) [inline INSTALL.md:68]: And at runtime, per previous code-signing comment change, above?

**Review by ctrlaltdavid — CHANGES_REQUESTED** [review]

File: INSTALL.md (hunk 1 above)
Comment (kasenvr) [reply INSTALL.md:68]: I don't believe runtime is necessary, runtime alone will not work. So if I put comments above, I should undo that. ;)

Comment (kasenvr) [reply INSTALL.md:68]: Oh, sorry, so I read it and that is correct... what I meant is the code signing certificate needs to be set when running CMake and when packaging as well. :)

Comment (ctrlaltdavid) [reply INSTALL.md:68]: Add something along the lines of, "as well as setting when running CMake"? And refer to how to set when running CMake?

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR640/Vircadia-Alpha-PR640-f04d7ab_8943032d.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR640/Vircadia-Alpha-PR640-f04d7ab.tar_8943032d.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR640/Vircadia-Alpha-PR640-f04d7ab.tar_8943032d.Z

**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR640/Vircadia-Alpha-PR640-f04d7ab_8943032d.dmg

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR640/Vircadia-Alpha-PR640-f04d7ab_8943032d.exe

**Review by ctrlaltdavid — APPROVED** [review]

**Pull request merged**